### PR TITLE
test

### PR DESCRIPTION
**Pull request by MvRemmerden**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Review by fracazo — CHANGES_REQUESTED** [review]
Review

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To start using Homebrew Cask, you just need [Homebrew](https://brew.sh/) install
 
 <img src="https://i.imgur.com/FNNM6WL.gif" width="547" alt="Installing and uninstalling Alfred (animated gif)">
 
-Slower, now:
+Slower, now:s
```
Comment (MvRemmerden) [inline README.md:17]:
```suggestion
Slower, now: sss
```

Fixed it, can you have a look again?

Comment (fracazo) [reply README.md:17]: > Fixed it, can you have a look again?

Sure adding this single comment but using ^ enter

**Review by fracazo — APPROVED** [review]
LGTM! Approved!

**Pull request closed without merging**